Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageObjectImpl.cpp

```diff
@@ -183,10 +183,9 @@ namespace ImageProcessingAtom
             return EAlphaContent::eAlphaContent_Absent;
         }
 
-        //if it's compressed format, return indeterminate. if user really want to know the content, they may convert the format to ARGB8 first
         if (!CPixelFormats::GetInstance().IsPixelFormatUncompressed(m_pixelFormat))
         {
-            AZ_Assert(false, "the function only works right with uncompressed formats. convert to uncompressed format if you get accurate result");
+            AZ_TracePrintf("Image processing", "GetAlphaContent() was called for compressed format\n");
             return EAlphaContent::eAlphaContent_Indeterminate;
```

Comment:
While I agree this should never be an assert since its a data error, should it be at least a warning?  What will this cause to happen if it appears?
- Trace - no difference in function or operation or visuals
- Warning - no difference in visuals but could impact performance, or some functionality
- Error - obvious difference in visuals and operation of the game, functionality broken

Reply:
It may happen if we use this function get get the alpha information with a source asset with compressed format (for example, come dds files). It doesn't do harm now since this won't be used this way anymore. 
Even it happened, the return value won't do harm.